MUXSDKStats/MUXSDKStats/MUXSDKPlayerBinding.m

```diff
@@ -117,6 +119,19 @@ - (BOOL) checkIfNotificationIsRelevant:(NSNotification *)notif {
     return notificationItem == _playerItem;
 }
 
+- (void)handleConnectionTypeDetected:(NSNotification *)notif {
+    dispatch_async(dispatch_get_main_queue(), ^{
```

Comment:
👍  That's right, we get the notification on a background thread but handle it on the main

Reply:
You may have already thought this through but is it possible we get this notification for a different player item than the current one? I seem to remember this could happen with some other async things and we sometimes use `checkIfNotificationIsRelevant`